apps/ui/src/views/Space.vue

```diff
@@ -33,7 +38,9 @@ watchEffect(() => {
 });
 
 watchEffect(() => {
-  if (!space.value) return setFavicon(null);
+  if (!space.value) {
+    return setFavicon(null);
+  }
```

Comment:
Maybe we should quite early like in `unOnmounted` as this is already handled for whitelabel in LayoutApp.